lib/msal-node/src/cache/distributed/DistributedCachePlugin.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountEntity, ICachePlugin, TokenCacheContext } from "@azure/msal-common";
+import { TokenCache } from "../TokenCache";
+import { IPartitionManager } from "./IPartitionManager";
+import { ICacheClient } from "./ICacheClient";
+
+export class DistributedCachePlugin implements ICachePlugin {
+    private client: ICacheClient;
+    private partitionManager?: IPartitionManager;
+
+    constructor(client: ICacheClient, partitionManager?: IPartitionManager) {
```

Comment:
When would a partition manager not be provided? Seems like it is needed for both functions.

Reply:
Yes, you are right, this should be required.